src/components/ebay-toggle-button-group/index.marko

```diff
@@ -0,0 +1,19 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = ["class", "columns"];
+
+<ul
+    class=["toggle-button-group", input.class]
+    data-columns=input.columns
+    ...processHtmlAttributes(input, ignoredAttributes)
+>
+    <for|{ layoutType = input.layoutType, ...buttonInput }, i| of=input.buttons>
```

Comment:
Good! I assume this removes layoutType in buttonInput and then we don't need to pass ignoredArrtibutes anymore. 
Maybe we should do something in other places. I like this approach here (and I cant stand the whole ignoredAttributes object)